libs/api-interfaces/src/lib/aggregate-views/view-models/resources/audio-item/multilingual-text-item.interface.ts

```diff
@@ -4,6 +4,8 @@ import { MultilingualTextItemRole } from './multilingual-text-item-role.enum';
 export interface IMultlingualTextItem {
     languageCode: LanguageCode;
 
+    defaultLanguageCode: LanguageCode;
```

Comment:
No- this does not belong here. You read `defaultLanguageCode` off the content config and it thus influences how you present a `MultilingualTextItem`. It doesn't affect the data itself. Please remove this.